File: test/fixtures/output/ruby/native/compression.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)
request["accept-encoding"] = 'deflate, gzip, br'

response = http.request(request)
puts response.read_body

File: test/fixtures/output/node/axios/compression.js
var axios = require("axios").default;

var options = {
  method: 'GET',
  url: 'http://mockbin.com/har',
  headers: {'accept-encoding': 'deflate, gzip, br'}
};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});